.github/workflows/main.yml

```diff
@@ -3,10 +3,10 @@ name: Claude Code Review
 on:
   pull_request:
     types: [opened, edited]
-  pull_request_review_comment:
-    types: [created, edited]
```

Comment:
The code patch is commenting out event triggers for pull request review comments and issue comments. This could potentially reduce the workflow's responsiveness to certain events. Consider if this change aligns with the intended behavior of the workflow.
          ```yaml
            # Consider keeping these events if they are still needed:
pull_request_review_comment:
  types: [created, edited]
issue_comment:
  types: [created, edited]
          ```